File: app/src/main/java/com/rockon999/android/leanbacklauncher/apps/InstallingLaunchPointListener.java
package com.rockon999.android.leanbacklauncher.apps;

public interface InstallingLaunchPointListener {
    void onInstallingLaunchPointAdded(LaunchPoint launchPoint);

    void onInstallingLaunchPointChanged(LaunchPoint launchPoint);

    void onInstallingLaunchPointRemoved(LaunchPoint launchPoint, boolean z);
}

File: app/src/main/java/com/amazon/tv/leanbacklauncher/apps/InstallingLaunchPointListener.kt
package com.amazon.tv.leanbacklauncher.apps

interface InstallingLaunchPointListener {
    fun onInstallingLaunchPointAdded(launchPoint: LaunchPoint?)
    fun onInstallingLaunchPointChanged(launchPoint: LaunchPoint?)
    fun onInstallingLaunchPointRemoved(launchPoint: LaunchPoint?, z: Boolean)
}